### PR TITLE
Adding new home component

### PR DESCRIPTION
**Pull request by wdhenarangoda**
Hey devdojo,

I'm adding a new component called 'build home'. Please accept it.
Thank you!

### CODE REVIEW

File: components/build-home.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+              </a>
+              <a href="#_" class="flex items-center px-6 py-3 text-gray-500 bg-gray-100 rounded-md hover:bg-gray-200 hover:text-gray-600">
+                Learn More
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="w-full md:w-1/2">
+          <div class="w-full h-auto overflow-hidden rounded-md shadow-xl sm:rounded-xl">
+              <img src="https://assets-news.housing.com/news/wp-content/uploads/2020/11/24190402/What-is-an-immovable-property-FB-1200x700-compressed.jpg" class="">
```
Comment (bobbyiliev) [inline components/build-home.html]: Would you mind changing the image with an image from https://unsplash.com/?

File: components/build-home.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            </div>
+        </div>
+      </div>
+    </div>
+  </section>
```
Comment (bobbyiliev) [inline components/build-home.html]: It looks like that there are some extra spaces throughout the file, would you mind formatting it?

**Review by bobbyiliev — CHANGES_REQUESTED** [review]
Looks great! Just a couple of remarks:

File: components/build-home.html (hunk 2 above)
Comment (wdhenarangoda) [reply components/build-home.html]: Done!

File: components/build-home.html (hunk 1 above)
Comment (wdhenarangoda) [reply components/build-home.html]: Done!

File: components/build-home.html (hunk 2 above)
Comment (wdhenarangoda) [reply components/build-home.html]: Please accept my pull request.

File: components/build-home.html (hunk 1 above)
Comment (wdhenarangoda) [reply components/build-home.html]: Please accept my pull request.

**Review by bobbyiliev — APPROVED** [review]

**Pull request merged**